pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -86,6 +109,27 @@ func readyProtocol(
 
 	switch ctx.Err() {
 	case context.DeadlineExceeded:
+		for _, memberID := range group.groupMemberIDs {
+			memberAddress, err := memberIDToAddress(memberID, publicKeyToAddressFn)
+			if err != nil {
+				logger.Errorf(
+					"could not convert member ID to address for a member of "+
+						"keep [%v]: [%v]",
+					group.groupID,
+					err,
+				)
+				continue
+			}
+			if !readyMembers[memberAddress] {
+				logger.Errorf(
+					"member [0x%v] has not announced its readiness for keep [%v]; "+
```

Comment:
For consistency, we should drop `0x` prefixing in the message as we did already in https://github.com/keep-network/keep-ecdsa/pull/654/commits/a43b3ab873000944f5b55e92f85154ac974393b9#diff-7dac81c8ac9c4245caf522776d5d4bb167a7012110db13c5a702b365b9b80f1aL64
```suggestion
					"member [%s] has not announced its readiness for keep [%s]; "+
```